src/components/MedicationGroup.spec.js

```diff
@@ -11,16 +12,46 @@ describe('MedicationGroup', () => {
           { id: 2, medName: 'Enalapril' },
           { id: 3, medName: 'Prednisolon' },
         ]}
+        editMode={false}
+        handleDeleteClick={jest.fn()}
+        handleEditClick={jest.fn()}
       />
     )
 
     const time = screen.getByRole('time')
     const medList = screen.getByRole('list')
     const meds = screen.getAllByRole('listitem')
-    const button = screen.getByRole('button', { name: '-' })
     expect(time).toBeInTheDocument()
     expect(medList).toBeInTheDocument()
     expect(meds).toHaveLength(3)
-    expect(button).toBeInTheDocument()
+  })
+
+  it('renders edit and delte button in edit mode', () => {
```

Comment:
Just a little Typo 😉
```suggestion
  it('renders edit and delete button in edit mode', () => {
```